probe/probe.go

```diff
@@ -582,6 +589,57 @@ func getAnnotatedPodLabels(m map[string]string) mapset.Set[string] {
 	return b
 }
 
+// Store container default posture data
+func setPostureData(buf []byte) error {
+	var kd *KubeArmorProbeData
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
+	err := json.Unmarshal(buf, &kd)
+	if err != nil {
+		log.Println("an error occured when parsing file", err)
+		return err
+	}
+
+	postureData = append(postureData, kd.ContainerDefaultPosture.FileAction)
+	postureData = append(postureData, kd.ContainerDefaultPosture.CapabilitiesAction)
+	postureData = append(postureData, kd.ContainerDefaultPosture.NetworkAction)
+
+	return nil
+}
+
+func getSecurityPostureAndVisibility(c *k8s.Client) (map[string][]string, error) {
+	// Namespace/host security posture and visibility setting
+	mp := make(map[string][]string)
+	namespaces, err := c.K8sClientset.CoreV1().Namespaces().List(context.Background(), metav1.ListOptions{})
+
+	if err != nil {
+		return mp, err
+	}
+
+	for _, ns := range namespaces.Items {
+
+		filePosture := postureData[0]
+		capabilityPosture := postureData[1]
+		networkPosture := postureData[2]
+
+		if len(ns.Annotations["kubearmor-file-posture"]) > 0 {
+			filePosture = ns.Annotations["kubearmor-file-posture"]
+		}
+
+		if len(ns.Annotations["kubearmor-capabilities-posture"]) > 0 {
+			capabilityPosture = ns.Annotations["kubearmor-capabilities-posture"]
+		}
+
+		if len(ns.Annotations["kubearmor-network-posture"]) > 0 {
+			networkPosture = ns.Annotations["kubearmor-network-posture"]
+		}
+
+		mp[ns.Name] = append(mp[ns.Name], "file("+filePosture+"), capabilities("+capabilityPosture+"), network("+networkPosture+")")
+		mp[ns.Name] = append(mp[ns.Name], ns.Annotations["kubearmor-visibility"])
```

Comment:
Can we used named fields?

```suggestion
		mp[ns.Name].DefaultPosture = "file("+filePosture+")+ capabilities("+capabilityPosture+"),+network("+networkPosture+")
		mp[ns.Name].Visibility = ns.Annotations["kubearmor-visibility"]
```